ksql-engine/src/test/resources/ToStruct.java

```diff
@@ -0,0 +1,22 @@
+package org.damian.ksql.udf;
+
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+
+/**
+ * Class used to test UDFs. This is packaged in udf-example.jar
+ */
+@UdfDescription(name = "tostruct", description = "converts things to a struct")
```

Comment:
```suggestion
@UdfDescription(name = "tostruct", description = "converts a string to a struct containing the string")
```
Alternatively, "wraps a string in a struct"?